interface/batchcom/batchEmail.php

```diff
@@ -1,60 +1,72 @@
 <?php
-/*    
-    batch Email processor, included from batchcom 
-*/
+/**
+ * Batch Email processor, included from batchcom
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  cfapress
+ * @author  Jason 'Toolbox' Oettinger <jason@oettinger.email>
+ * @copyright Copyright (c) 2008 cfapress
+ * @copyright Copyright (c) 2017 Jason 'Toolbox' Oettinger <jason@oettinger.email>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
 
 // create file header.
 // menu for fields could be added in the future
+require_once("../globals.php");
+use OpenEMR\Core\Header;
 
 ?>
 <html>
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<link rel="stylesheet" href="batchcom.css" type="text/css">
+    <title><?php echo xlt('Email Notification Report'); ?></title>
+    <?php Header::setupHeader(); ?>
 </head>
-<body class="body_top">
-<span class="title"><?php xl('Batch Communication Tool','e')?></span>
-<br><br>
-<span class="title" ><?php xl('Email Notification Report','e')?></span>
-<br><br>
+<body class="body_top container">
+<header class="row">
+    <?php require_once("batch_navigation.php");?>
+    <h1 class="col-md-12">
+        <a href="batchcom.php"><?php echo xlt('Batch Communication Tool')?></a>
+        <small><?php echo xlt('Email Notification Report'); ?></small>
+    </h1>
+</header>
+<main class="row">
+    <ul class="col-md-12">
+        <?php
+        $email_sender = $_POST['email_sender'];
+        $sent_by = $_SESSION["authId"];
+        $msg_type = xlt('Email from Batchcom');
```

Comment:
Above variable isn't used in this script (minor; I'll remove this after bring this code into the codebase)